GSA-TTS/federal-website-standards
# Add the GSA logo to the footer

## Context
Fixes #223

## Description
This wraps our footer content in an actual footer element and then adds the GSA logo to the footer.

## How to verify this change
1. Visit the preview URL and scroll to the bottom. Observe the GSA logo. The logo should appear next to the two `<p>` lines of text on desktop/tablet width and on its own line when viewed in a mobile viewport size.
